code/datums/weather/weather_types/radiation_storm.dm

```diff
@@ -30,44 +30,46 @@
 
 /datum/weather/rad_storm/weather_act(mob/living/L)
 	var/resist = L.getarmor(null, "rad")
-	if(prob(40))
-		if(ishuman(L))
-			var/mob/living/carbon/human/H = L
-			if(H.dna && !(HAS_TRAIT(H, TRAIT_RADIMMUNE) || HAS_TRAIT(H, TRAIT_MUTATEIMMUNE))) // NSV13 Don't add mutations for species that are trait immune, IPCs
-				if(prob(max(0,100-resist)))
-					H.randmuti()
-					if(prob(50))
-						if(prob(90))
-							H.easy_randmut(NEGATIVE+MINOR_NEGATIVE)
-						else
-							H.easy_randmut(POSITIVE)
-						H.domutcheck()
-			if( HAS_TRAIT( H, TRAIT_IPCRADBRAINDAMAGE ) ) // NSV13 IPCs will gain brain damage instead of mutations when exposed to radiation
-				if(prob(max(0,100-resist)))
-					if(prob(50))
-						var/trauma_type = pickweight( list(
-							BRAIN_TRAUMA_MILD = 65,
-							BRAIN_TRAUMA_SEVERE = 30,
-							BRAIN_TRAUMA_SPECIAL = 5
-						) )
-						var/resistance = pick(
-							95;TRAUMA_RESILIENCE_BASIC,
-							// 30;TRAUMA_RESILIENCE_SURGERY,
-							// 15;TRAUMA_RESILIENCE_LOBOTOMY,
-							5;TRAUMA_RESILIENCE_MAGIC
-						)
-						H.gain_trauma_type( trauma_type, resistance )
+	if(L.radiation >= 1000)
```

Comment:
```suggestion
	if(L.radiation >= 1000) //NSV start, puts a maximum on how irradiated you can get
```

Reply:
yeah that makes sense, good call. commited via the lil button it gave me here. super useful!